beitragsordnung.md

```diff
@@ -1,39 +1,58 @@
+        | Partner/Familie (im gleichen Haushalt wie Mitglied) | 7,50 Euro | - |
+        | Jugendliche (ab vollendetem 16. Lebensjahr) | 15 Euro | - |
+        | Senioren (ab vollendetem 65. Lebensjahr) | 15 Euro | Ab 7,50 Euro |
 
-    | Art der Mitgliedschaft  | Mitglieder mit Stimmrecht | Fördermitglieder |
-    | ----------------------- | ------------------------- | ---------------- |
-    | Erwachsene (ab vollendetem 18. Lebensjahr) | 25 Euro | Ab 12,50 Euro |
-    | Partner/Familie (im gleichen Haushalt wie Mitglied ) | 7,50 Euro | - |
-    | Jugendliche (ab vollendetem 14. Lebensjahr) | 15 Euro | - |
-    | Kinder (bis zum vollendeten 14. Lebensjahr) | 7,50 Euro | - |
-    | Senioren (ab vollendetem 65. Lebensjahr) | 15 Euro | Ab 7,50 Euro |
+        Der vierteljährliche, halbjährliche und jährliche Beitrag berechnet sich entsprechend durch Multiplikation der Monatsbeiträge mit der Anzahl der Monate (4, 6 oder 12).
 
-5. Für die Nutzung der Räumlichkeiten des Vereins entrichten ordentliche Mitglieder mit Stimmrecht eine reduzierte Tagespauschale von 3 Euro. Alle anderen Personen (Gäste) entrichten pro Nutzungstag innerhalb der regulären Öffnungszeiten pauschal mindestens 10 Euro.
 
-6. Alle Besucher (Mitglieder und Gäste) können eine Begleitperson mit in die Räumlichkeiten bringen, sofern diese am gleichen Projekt mitarbeitet. Die Begleitperson eines Mitglieds entrichtet eine Tagespauschale von 3 Euro. Die Begleitperson eines Nicht-Mitglieds entrichtet eine Tagespauschale von 8 Euro.
+2. Tagespauschale
+    1. Für die private Nutzung der Räumlichkeiten des Vereins entrichten ordentliche Mitglieder mit Stimmrecht eine reduzierte Tagespauschale von 3 Euro. Gäste entrichten pro Nutzungstag innerhalb der regulären Öffnungszeiten pauschal 10 Euro.
 
-7. Der Verein erhebt eine Aufnahmegebühr von 10 Euro, die nach Aufnahme in den Verein fällig wird und mit dem ersten Mitgliedsbeitrag zusammen überwiesen werden kann oder gemeinsam mit dem ersten Mitgliedsbeitrag im Lastschriftverfahren eingezogen wird.
+    2. Alle Besucher (Mitglieder und Gäste) können eine Begleitperson mit in die Räumlichkeiten bringen, sofern diese am gleichen Projekt mitarbeitet. Die Begleitperson eines Mitglieds entrichtet eine Tagespauschale von 3 Euro. Die Begleitperson eines Nicht-Mitglieds entrichtet eine Tagespauschale von 8 Euro.
 
-8. Bei Nutzung des Lastschriftverfahrens und Unregelmäßigkeiten auf den Konten des Vereinsmitglieds – z.B. mangelnde Deckung, Änderung der Kontonummer, der Anschrift, des Namens u.a. ohne Information an den Verein – werden pro Mahnung etwaige Rücklastschriftgebühren und eine Mahngebühr von 2,50 Euro erhoben.
+    3. Mitglieder können zur Entrichtung der Tagespauschale eine Zehnerkarte zum Preis von zehn Tagespauschalen (aktuell 10 x 3 Euro also 30 Euro) erwerben. Sie erlaubt dem Mitglied die zehnmalige Nutzung der Räumlichkeiten – auch wenn sich die Höhe der Tagespauschale im Nachhinein ändert. 
```

Comment:
Vorschlag von Thorben:
```suggestion
    4. Mitglieder können zur Entrichtung der Tagespauschale eine Zehnerkarte zum Preis von zehn Tagespauschalen (aktuell 10 x 3 Euro also 30 Euro) erwerben. Sie erlaubt dem Mitglied die zehnmalige Nutzung der Räumlichkeiten – auch wenn sich die Höhe der Tagespauschale im Nachhinein ändert.
    
    5. Mitglieder die Arbeitsleistungen für die Werkstatt erbringen (mind. eine Stunde) müssen am selben Tag für die private Nutzung keine Tagespauschale leisten. Dabei erfolgt eine Eintragung im Kassenbuch mit dem Vermerk "Arbeitsleistung". 
```

Reply:
Bin mir unsicher bezüglich zwei Aspekten:
- "eine Stunde": Würde das etwas runterschrauben. 45 Minuten vielleicht?
- Eintrag im Kassenbuch: Würde ich weglassen. Einträge mit 0€ machen das Kassenbuch unübersichtlich. Finde es ausreichend an die Eigenverantwortung der Mitglieder zu appellieren.

Reply:
Finde eine Stunde als Festsetzung erstmal gut. Man kann das ja mit Augenmaß handhaben also auch mal aufrunden. Letzteres wäre für mich auch in Ordnung

Reply:
Sollten wir noch spezifizieren, dass man das dann natürlich nicht noch zusätzlich als Arbeitsleistung für die Pflichtstunden einreichen kann?